app/controllers/IvrController.scala

```diff
@@ -22,17 +22,20 @@ import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
 
 import scala.concurrent.Future
+import views.html.CUIViews.AskHmrcOnlineCUIView
```

Comment:
@sanjayshukla2205 do we still need this import? If not can we remove it and the injected view

Reply:
Hi Lorenzo, Yes this is no more required, hence i am removing it.